src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -187,12 +182,33 @@ internal override ImmutableArray<string> GetAppliedConditionalSymbols()
             internal override IEnumerable<SecurityAttribute> GetSecurityInformation()
                 => Array.Empty<SecurityAttribute>();
 
-            internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => false;
+            internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => !this.IsOverride;
 
-            internal override bool IsMetadataVirtual(bool ignoreInterfaceImplementationChanges = false) => false;
+            internal override bool IsMetadataVirtual(bool ignoreInterfaceImplementationChanges = false) => this.IsOverride;
 
             internal override bool SynthesizesLoweredBoundBody => true;
 
+            internal abstract override void GenerateMethodBody(TypeCompilationState compilationState, DiagnosticBag diagnostics);
+        }
+
+        private sealed class GetAccessorSymbol : AccessorSymbol
+        {
+            public override string Name { get; }
+
+            public GetAccessorSymbol(SynthesizedRecordPropertySymbol property, string paramName) : base(property)
+            {
+                Name = SourcePropertyAccessorSymbol.GetAccessorName(
+                    paramName,
+                    getNotSet: true,
+                    isWinMdOutput: false /* unused for getters */);
+            }
+
+            public override MethodKind MethodKind => MethodKind.PropertyGet;
+
+            public override TypeWithAnnotations ReturnTypeWithAnnotations => _property.TypeWithAnnotations;
+
+            public override ImmutableArray<ParameterSymbol> Parameters => _property.Parameters; // PROTOTYPE: Shouldn't the parameters be owned by this symbol?
```

Comment:
>// PROTOTYPE: Shouldn't the parameters be owned by this symbol? [](start = 96, length = 63)

Yes, they should be. I think [https://github.com/dotnet/roslyn/pull/44883](https://github.com/dotnet/roslyn/pull/44883) addresses that.

Reply:
Will merge [https://github.com/dotnet/roslyn/pull/44883](https://github.com/dotnet/roslyn/pull/44883) when that has been committed.

---
In reply to: [443269323](https://github.com/dotnet/roslyn/pull/45336#discussion_r443269323) [](ancestors = 443269323)